### DISCUSSION TITLE
各位大佬，请教一下ElectronBot-Studio的源码是用的什么开发工具写的啊

### DISCUSSION THREAD

**Discussion by zp-95** [peng-zhihui/ElectronBot#257] [Q&A]
各位大佬，请教一下ElectronBot-Studio的源码是用的什么开发工具写的啊，新人不太懂，想复刻学习一下电子和它的上位机软件。

**Comment by Kitem** [comment]
您好，我是焕挺，您的邮件已经收到，我会尽快回复！

**Comment by VendettaMask** [comment]
来信已收，谢谢！

**Comment by ZK-y53** [comment]
Clion
















在 2024-05-13 16:16:53，"zp-95" ***@***.***> 写道：

各位大佬，请教一下ElectronBot-Studio的源码是用的什么开发工具写的啊，新人不太懂，想复刻学习一下电子和它的上位机软件。

**Reply by zp-95** [reply]
Clion好像不直接支持c#吧，是不是要配置一下，有配置方法吗大佬

**Comment by jeongyeham** [comment]
如果您对Jetbrains的软件感兴趣，请使用Rider开发unity的部分。
